packages/sanity/src/structure/components/pane/PaneMenuButtonItem.tsx

```diff
@@ -108,6 +108,7 @@ function PaneContextMenuItem(props: {disabled?: boolean; node: _PaneMenuItem}) {
         pressed={node.selected}
         text={title}
         tone={node.tone}
+        data-testid={`action-${node.title?.toLocaleLowerCase().replace(' ', '')}`}
```

Comment:
should this also be `replaceAll` instead of single `replace`?

Reply:
Ah yes, it should! Good catch! 
I'll fix this!

Reply:
Small related thought: would it make sense to move this to a tiny helper function? I noticed it's repeated a few times, and having the code inline like this doesn't make it _super_ clear to me how/why the node title -> test id transformation works.

Reply:
sure, that makes sense :)

Reply:
Done!